src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -600,9 +605,9 @@ public bool SignOut()
         {
             return RequestSignOut();
         }
-        public void CloseWindow()
+        public void CloseWindow(bool isCheckboxChecked)
```

Comment:
Are any of these API used in Dynamo Revit repo?

Reply:
Also instead of passing param, any other way to do it, e.g. static boolean saved?

Reply:
> Are any of these API used in Dynamo Revit repo?
The APIs are not used in DynamoRevit